src/flashbar/__tests__/flashbar.test.tsx

```diff
@@ -6,29 +6,9 @@ import Flashbar, { FlashbarProps } from '../../../lib/components/flashbar';
 import Button from '../../../lib/components/button';
 import createWrapper from '../../../lib/components/test-utils/dom';
 import styles from '../../../lib/components/flashbar/styles.css.js';
-import { createFlashbarWrapper, findList } from './common';
+import { createFlashbarWrapper, findList, testFlashDismissal } from './common';
 import { DATA_ATTR_ANALYTICS_FLASHBAR } from '../../../lib/components/internal/analytics/selectors';
-
-let mockUseAnimations = false;
```

Comment:
Simplified disabling motion in test by using our own official method instead of mocking

Reply:
Nice